src/extensions/default/NavigationAndHistory/html/recentfiles-template.html

```diff
@@ -0,0 +1,165 @@
+<style>
```

Comment:
I kept it purposefully in the template as it affects the DOM only when appended, to minimize any unintentional style override.